app/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser.rb

```diff
@@ -69,23 +88,43 @@ def parse_firmware(firmware, uuid)
       }
     end
 
+    def parse_hardware(node)
```

Comment:
The data that we need parse are: total memory in `node.memoryModules`, total CPU cores and the maximum processor frequency in ` node.processors`, as you can see in the method. It was decided during a meeting with Lenovo guys.